app/src/main/java/emu/skyline/data/DataItem.kt

```diff
@@ -39,6 +39,11 @@ data class AppItem(private val meta : AppEntry) : DataItem() {
      */
     val uri get() = meta.uri
 
+    /**
+     * The string used as the sub-title, we currently use the author
```

Comment:
```suggestion
     * The string used as the application version, we currently use the author
```

Reply:
Drop the part after the comma and you're good.